src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+(if: $oldSchoolJuggalosSeen is false)[>>[[Some old-school juggalos in a heated conversation by the side of the building|The Gutter - Outside - Old-school juggalos]]]
+(if: $golfCartSeen is false)[>>[[Ron’s shitheap golf cart|The Gutter - Golf cart]]]
+(if: $jarSeen is false)[>>[[The jar|The Gutter - Jar]]]
+
+::The Gutter - Outside - Old-school juggalos
+A tall, chubby juggalo stroking a long beard and a shorter, musclebound ninja with green Kool-Aid dyed hair were inches away from each other, shouting in agreement as they chainsmoked:
 
-I picked [[the least muddy trail towards civilization|The Gutter - Outside - Approach]].
+“Sorry to be all deep and serious, but I got a lotta shit on my mind.”
 
-::The Gutter - Outside - Approach <1390,73>
-My body and mind were screaming as [one]<trigger1|.
+“It breaks my heart dude, but it had to be said. Ninjas today just don’t know.”
 
-|section2)[I kept my head down and my hood up. I used all of my energy to move forward through the tarp-covered alleyways of Gatherville.
+“They don’t, man! The Dark Carnival saved my life, dude, and it’s about love. Shit is about family, dude. If these ninjas were out there listening to the shit...”
 
-My gut lurched as I realized I’d need to ask around for more information on Anna. The last thing I wanted to do was [talk to anyone]<trigger2|.]
+“Listen to the //shit//!!”
 
-|section3)[My brain bounced into the darkness of reality, searching for answers. It was still pretty early in the night. Around this time, anyone who knew anything was
+(click: "Listen to the shit")[$dissolve[The shorter ninja started jumping and kicking an unknown assailant at this point. Large picked up on the energy and took his hands away from his beard. He started punching one hand into the other to punctuate his points.
 
-[[>>Getting fucked up at the Gutter|The Gutter - Outside - Intro]]
-[[>>Looking to score at the Roach Market|Market scene]]
-[[>>Catching an early show at the Church|Church scene]]]
+“Exactly! If they were listening; like Dark Lotus? '//Juggalo Family//'? That’s my real shit. That’s what I’m about.”
 
-(click: ?trigger1)[(show: ?section2)]
-(click: ?trigger2)[(show: ?section3)]
+“It’s all about the money these days. Ninjas out there got on 4,000-dollar hatchet man jerseys and they think they the shit.”]]
 
-::Market scene <1874,87>
-Double-click this passage to edit it.
+(click: "they think they the shit")[$dissolve[“Had a hole in my shirt the other day and I walk by one of them fancy-ass ninjas. You know what he said?”
 
-::Church scene <2120,91>
-Double-click this passage to edit it.
+“Naw, man. No way! It ain’t about the clothes!”
 
-::The Gutter - Outside - Intro <1262,192>
-I stood outside a weathered, flat-roofed wooden building with a flickering [neon sign]<trigger1|.
+“He called me a juggahoe. You heard that before? I laughed in his face. I told him it was all family, but I’d beat his ass if that was how he needed to be educated.”
```

Comment:
It might be fun to italicize the "hoe" in that word for emphasis:

"He called me a jugga_hoe_. You heard that before? I laughed in his face."

Reply:
Totally! Makes sense to me.